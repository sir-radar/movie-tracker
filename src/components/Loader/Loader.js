import React from 'react';

const Loader = () => {
  return (
    <section className='loading mt-5 mx-auto'>
      <i className="fa fa-spinner fa-5x fa-spin"></i>
    </section>
  );
}

export default Loader;
