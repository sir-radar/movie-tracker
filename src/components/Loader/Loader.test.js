import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loader from './Loader';
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer"

afterEach(cleanup)

const renderComponent = () => {
  const div = document.createElement("div");
  return render(<Loader/>, div)
}

it("renders without crashing", () => {
  renderComponent()
})


it("matches snapshot", () => {
  const tree = renderer.create(<Loader/>).toJSON();
  expect(tree).toMatchSnapshot();
})