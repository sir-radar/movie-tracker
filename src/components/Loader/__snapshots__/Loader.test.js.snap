// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<section
  className="loading mt-5 mx-auto"
  data-testid="loader"
>
  <i
    className="fa fa-spinner fa-5x fa-spin"
  />
</section>
`;
